sys/bexpr/src/StautDef.hs

```diff
@@ -27,51 +27,65 @@ import           ProcDef
 import           ProcId
 import           SortId
 import           StatId
-import           VarEnv
 import           ValExpr
+import           VarEnv
 import           VarId
 
 -- | combine parameters of stautdef, state identifier, and stautdef variables into single parameter list
 -- must be consistent with @combineArguments@
-combineParameters :: [VarId] -> VarId -> [VarId] -> [VarId] 
+combineParameters :: [VarId] -> VarId -> [VarId] -> [VarId]
 combineParameters params stateId vars' = params ++ (stateId:vars')
 
 -- | combine arguments of stautdef instantiation, state identifier, and stautdef variables into single argument list
 -- must be consistent with @combineParameters@
-combineArguments :: [VExpr] -> VExpr -> [VExpr] -> [VExpr] 
+combineArguments :: [VExpr] -> VExpr -> [VExpr] -> [VExpr]
 combineArguments params stateId vars' = params ++ (stateId:vars')
 
-translate :: Map.Map FuncId (FuncDef VarId) -> Id -> Id -> Name -> [ChanId] -> [VarId] -> ExitSort -> [StatId] -> [VarId] -> [Trans] -> StatId -> VEnv -> (ProcDef, BExpr)
+translate
```

Comment:
@pjljvandelaar could you document the other parameters? (and maybe improve the documentation I added :) )

Reply:
Ok